app/features/analysis/calculate/tags/SelectTagsModal.js

```diff
@@ -140,8 +142,17 @@ class SelectTagsModal extends Component {
 
     // TODO: grab the blue color for cancel from a global stylesheet
     _renderNavigation() {
-        if (Platform.OS === 'ios') {
+        if (Platform.OS === 'ios' && !Device.isiPhoneX()) {
             var statusBar = (<View style={{height: 20, width: 9001, backgroundColor: 'black'}}></View>);
+        } else if (Device.isiPhoneX()) {
```

Comment:
make it `if device is iphonex else if platform == ios` to simplify logic so you don't do that extra check